build/Helix/InstallTestAppDependencies.ps1

```diff
@@ -1,7 +1,17 @@
-foreach($line in Get-Content ..\MUXControlsTestApp.dependencies.txt) {
-    Add-AppxPackage ..\$line
+function Install-Dependency
+{
+    Param($dependencyFilePath)
+
+    if(Test-Path $dependencyFilePath)
+    {
+        foreach($line in Get-Content $dependencyFilePath) 
+        {
+            Add-AppxPackage ..\$line
+        }
+    }
 }
 
-foreach($line in Get-Content ..\IXMPTestApp.dependencies.txt) {
-    Add-AppxPackage ..\$line
-}
\ No newline at end of file
+Install-Dependency ..\MUXControlsTestApp.dependencies.txt
+Install-Dependency ..\IXMPTestApp.dependencies.txt
```

Comment:
My assumption was that we'd invoke helix once for the dev project and then twice for the nuget project. In other words i'm surprised to see this script updated to include both dev project dependencies *and* nuget package test dependencies.

Reply:
Moved nuget test dependencies to a new InstallNugetTestAppDependencies.ps1